packages/frontend/src/lib/table/environment/ColumnName.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+  import type { EnvironmentState } from "@shared/src/models/IEnvironmentState";
+  import type { ModelInfo } from "@shared/src/models/IModelInfo";
+  export let object: EnvironmentState;
+</script>
+
+<div class="text-sm text-gray-700">
+  {object.recipeId}
```

Comment:
Can't we display the state as well ?

Reply:
I'm waiting the design from Mo to work on the content here